app/settings/users/users-edit.html

```diff
@@ -41,7 +41,7 @@ <h1 class="mode-context-title" ng-show="!user.id" translate>app.add_user</h1>
             <form name="form">
                 <div class="form-sheet">
                     <div class="form-field" ng-class="{ 'error': form.full_name.$invalid && form.full_name.$dirty }">
-                        <label translate>user.full_name</label>
+                        <label>{{'user.full_name' | translate}} <span style="color:red">*</span></label>
```

Comment:
Thank you for working on this 👍. There is a class for adding the red asterisk, check out how this is handled for the title and description in a post here: https://github.com/ushahidi/platform-client/blob/develop/app/main/posts/modify/post-value-edit.html#L7. We want to avoid inline-styles as far as possible :).